src/main/webapp/app/course/dashboards/assessment-dashboard/assessment-dashboard.component.html

```diff
@@ -2,7 +2,6 @@
     <div class="row justify-content-between">
         <div class="col-8">
             <h2>
-                <fa-icon [icon]="'arrow-left'" (click)="back()" class="back-button mr-2"></fa-icon>
```

Comment:
The `back()` method in the `.component.ts` file is now unused.

Reply:
Whoops, thanks for pointing that out!